### ISSUE TITLE
Keyword arguments for options

### ISSUE THREAD

**Issue by asppsa** [moneta-rb/moneta#175]
Moneta's convention of using a hash as the last argument for most methods in order to pass in options [has been deprecated in Ruby 2.7](https://rubyreferences.github.io/rubychanges/2.7.html#keyword-argument-related-changes), and won't be supported at all in 3.0.  As such, we should change the Moneta spec and the code to use keyword arguments for options instead.

**Comment by terceiro** [comment]
so there are problems in the code as well in ruby3.0? I was thinking it was just a test suite issue

**Comment by asppsa** [comment]
Yes, it's a problem across the package at present - you'll see a lot of `options = {}` at the end of function definitions.  We'll need to go through and change these all to `**options` and then work out what breaks
